cmd/vclusterctl/cmd/platform/start.go

```diff
@@ -142,3 +149,31 @@ func (cmd *StartCmd) Run(ctx context.Context) error {
 
 	return start.NewLoftStarter(cmd.Options).Start(ctx)
 }
+
+func ensureEmailProvided(email string) string {
+	for answerErr := validateEmail(email); answerErr != nil; {
+		email, answerErr = survey.NewSurvey().Question(&survey.QuestionOptions{
+			Question:       "Please specify an email address for the admin user",
+			ValidationFunc: validateEmail,
+		})
+	}
+
+	return email
+}
+
+func validateEmail(email string) error {
+	ret, err := verifier.Verify(email)
+	if err != nil {
+		return err
+	}
+
+	if ret.Disposable {
+		return fmt.Errorf(`disposable address "%s" not allowed`, email)
+	}
+
+	if !ret.HasMxRecords {
+		return fmt.Errorf(`address "%s" without MX record not allowed`, email)
+	}
+
+	return nil
+}
```

Comment:
This will perform SMTP or MX lookups from a CLI, potentially blocking folks in air-gapped or restricted networks from spinning up the platform.

I'd expect this logic in the platform code if connections can be established.
The CLI should only perform a quick sanity check using a regex or comparable mechanics.